src/vstest.console/TestPlatformHelpers/TestRequestManager.cs

```diff
@@ -664,11 +664,13 @@ static Architecture TranslateToArchitecture(PlatformArchitecture targetArchitect
                     return Architecture.ARM;
                 case PlatformArchitecture.ARM64:
                     return Architecture.ARM64;
+                case PlatformArchitecture.S390x:
+                    return Architecture.S390x;
                 default:
                     break;
             }
 
-            throw new TestPlatformException($"Invalid target architecture '{targetArchitecture}'");
+            return Architecture.Default;
```

Comment:
Please comment on why this is needed, and why we reverted back from throwing error, so we avoid having such back and forth in the future.

Reply:
done

Reply:
Is it worth adding some log here so that we can track record of invalid calls?

Reply:
done

Reply:
@cvpoienaru or telemetry even :)